src/Workspaces/Core/Portable/Shared/Extensions/SemanticModelExtensions.cs

```diff
@@ -170,6 +170,19 @@ private static ISymbol MapSymbol(ISymbol symbol, ITypeSymbol type)
                 }
             }
 
+            if (symbol is IMethodSymbol methodSymbol &&
+                methodSymbol.MethodKind == MethodKind.BuiltinOperator)
+            {
+                var comparer = SymbolEquivalenceComparer.Instance.ParameterEquivalenceComparer;
+
+                // see if we can map the built-in language operator to a real method
+                // on the containing type of the symbol.
```

Comment:
I feel like this comment needs some explanation here, since it's not obvious to me why this doesn't end up with the symbol we started with, especially since there's a FirstOrDefault going on here.

Reply:
(I'm guessing this is because System.String has both a metadata _and_ a build-in operator? But won't GetMembers return the one we started with?)

Reply:
Will comment.  However, the crux of it "the builtin operators are not returned in the members list of the type they're contained in".  Effectively, the compiler synthesizes them, points their 'containing type' a something like System.String, but otherwise does not make them reachable from teh other direction :)